src/library/Library.js

```diff
@@ -9,78 +10,129 @@ import {
 import Header from '../Components/Header';
 import FilterGraphic from '../../assets/filterOutline.svg';
 import LessonPlanButton from './components/LessonPlanButton';
+import LessonPlanService from '../services/LessonPlanService';
 
-const fillerLP = [
-  {
-    name: 'Jan 1, 2023',
-    isFavorited: false,
-    lastEdited: 'Jan 1, 2023',
-  },
-  {
-    name: 'Jan 2, 2023',
-    isFavorited: true,
-    lastEdited: 'Jan 2, 2023',
-  },
-  {
-    name: 'Jan 3, 2023',
-    isFavorited: true,
-    lastEdited: 'Jan 3, 2023',
-  },
-  {
-    name: 'Jan 4, 2023',
-    isFavorited: false,
-    lastEdited: 'Jan 4, 2023',
-  },
-];
+async function favouritePlan(name) {
```

Comment:
is it possible for you to remove these here and just do it inside handleFavouriteChange via the following Promise syntax instead?
```
LessonPlanService.favouriteLessonPlan(name)
    .then(() => {
        // do something here 
    })
    .catch((err) => {
        console.error(blablablabla)
    }))
```

Reply:
Hi it should be fixed now!

Reply:
perfect!!!